### DISCUSSION TITLE
Renovate fails to update Maven dependency from custom Maven repository

### DISCUSSION THREAD

**Discussion by wiebeck** [renovatebot/renovate#17678] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Not entirely sure if this is a bug or a missing feature. Adding a simple dependency (`install4j-runtime`) and a custom Maven repository to my Gradle build works fine for building the artifact but Renovate cannot update this dependency:

> Renovate failed to look up the following dependencies: com.install4j:install4j-runtime

The custom Maven repository is merely a static website hosting some jars and poms but files like `maven-metadata.xml' are missing. Maybe it's just not possible for Renovate to analyze such a dependency?

I have provided a minimal reproduction repository here: https://github.com/wiebeck/renovate-install4j-bug

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Looking up com.install4j:install4j-runtime in repository https://maven.ej-technologies.com/repository/
DEBUG: No concurrency limits
{
  "host": "maven.ej-technologies.com"
}
DEBUG: Content is not found for Maven url
{
  "url": "https://maven.ej-technologies.com/repository/com/install4j/install4j-runtime/maven-metadata.xml"
}
DEBUG: Failed to look up dependency com.install4j:install4j-runtime (com.install4j:install4j-runtime)(packageFile="app/build.gradle.kts", dependency="com.install4j:install4j-runtime")
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by viceice** [comment, marked as answer]
not a bug,you can see in logs that renovate needs the metadata file

**Reply by wiebeck** [reply]
Alright, thanks. That's what I assumed. Luckily EJ Technologies was so kind and now provide a `maven-metadata.xml` file for their lib: https://maven.ej-technologies.com/repository/com/install4j/install4j-runtime/maven-metadata.xml \o/
